misk-hibernate-testing/src/main/kotlin/misk/jdbc/VitessScaleSafetyChecks.kt

```diff
@@ -46,6 +54,186 @@ internal data class Variables(
   val QueriesProcessed: Map<String, Int>
 )
 
+/**
+ * A MySQL query explanation.
+ */
+internal class Explanation {
```

Comment:
The filename is `VitessScaleSafetyChecks` but I think this works on MySQL too?

Reply:
Yes in theory but the currently implementation is coupled to the way MySQL runs when we're using Vitess.